recipes/spectrum/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.6.4" %}
+{% set name = "spectrum" %}
+{% set sha256 = "7d8e3fe074b2026f61b6539c4f847d0cb1dbed41d5c6788f6100bb5884aa65ec" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/43/a4/c85957430db53744fd70ce726d8cd0000ee33e18da39a8132ca4e5cdb2f5/{{ name }}-{{ version }}.tar.gz
```

Comment:
Can you use the following canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18